neuro_san/test/unittest/unit_test_assert_forwarder.py

```diff
@@ -189,3 +189,47 @@ def assertLessEqual(self, first: Any, second: Any, msg: str = None):
         :param msg: optional string message
         """
         self.test_case.assertLessEqual(first, second, msg=msg)
+
+    # pylint: disable=invalid-name
+    def assertGist(self, gist: bool, acceptance_criteria: str, text_sample: str, msg: str = None):
+        """
+        Assert that the gist is true
+
+        :param gist: Pass/Fail value of the gist expected to be True
+        :param acceptance_criteria: The value to verify against
+        :param text_sample: The value appearing in the test sample
+        :param msg: optional string message
+        """
+        if msg is None:
+            msg = f"""
+text_sample unexpectedly did not match acceptance criteria.
+
+text_sample:
+{text_sample}
+
+acceptance_criteria:
+{acceptance_criteria}
+"""
+        self.test_case.assertTrue(gist, msg=msg)
```

Comment:
Unit tests need to assert that the gist value is true/false with its own messaging.